en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/service-discovery/choreo-connect-and-k8s.md

```diff
@@ -0,0 +1,48 @@
+# Configure Choreo Connect with Consul service mesh deployed on Kubernetes
+Example:
+
+![service discovery](../../../../assets/img/deploy/consul-reference-k8s.png)
+
+## Step 1 - Deploy Consul on K8s
+Deploy Consul using the [official Helm Chart](https://helm.releases.hashicorp.com/).<br>
+For more instructions, see the [Consul official documentation](https://www.consul.io/docs/k8s).
+## Step 2 - Modify Choreo Connect k8s artifacts 
+Add the `'consul.hashicorp.com/connect-inject': 'true'` annotation to Adapter's deployment spec
```

Comment:
```suggestion
Add the `'consul.hashicorp.com/connect-inject': 'true'` annotation to the Adapter's deployment specification.
```
Shall we add which file exactly we are meant to modify here?

Reply:
fixed 364f6cb